sigstore/_cli.py

```diff
@@ -245,6 +250,13 @@ def _parser() -> argparse.ArgumentParser:
         default=os.getenv("SIGSTORE_SIGNATURE"),
         help="The signature to verify against; not used with multiple inputs",
     )
+    input_options.add_argument(
+        "--bundle",
+        metavar="FILE",
+        type=Path,
+        default=os.getenv("SIGSTORE_BUNDLE"),
```

Comment:
Same thing here, we should refer to this as a REKOR_BUNDLE

Reply:
Sounds good! We're using the `SIGSTORE_` prefix everywhere, so this will become `SIGSTORE_REKOR_BUNDLE` under the current naming scheme 😅